enzy_htp/_interface/rosetta_interface.py

```diff
@@ -155,35 +174,62 @@ def action_on_wt(self, stru: Structure) -> Structure:
         
         Returns:
             return a copy of the changed structure."""
-        result = copy.deepcopy(stru)
+        stru = copy.deepcopy(stru)
+        # 0. remove ligands
+        remove_non_peptide(stru)
+        # TODO deal with NCAA
+        
         # 1. relax
-        self.parent_interface.relax(result, nstruct=20, )
+        fs.safe_mkdir(self.work_dir)
+        relax_dir = f"{self.work_dir}/wt_relax/"
+        score_df = self.parent_interface.relax(
```

Comment:
Use this for now but in the future I have developed a RosettaScripts based `minimize` function that will handle this functionality. Also a `RosettaScriptsProtocol` I guess if you want to use MPI it is necessary to use the commadline, but otherwise it is not that important and you can really speed up the run time using constraints.

Reply:
Good to know!